scripts/sfpshow

```diff
@@ -391,15 +393,18 @@ class SFPShow(object):
         output = ''
 
         sfp_info_dict = state_db.get_all(state_db.STATE_DB, 'TRANSCEIVER_INFO|{}'.format(interface_name))
-        output = interface_name + ': ' + 'SFP EEPROM detected' + '\n'
-        sfp_info_output = self.convert_sfp_info_to_output_string(sfp_info_dict)
-        output += sfp_info_output
+        if sfp_info_dict['type'] == RJ45_PORT_TYPE:
```

Comment:
the RJ45 is only by Nvidia? no other vendors are using it?

Reply:
although some vendors do have SKU with RJ45 ports but didn't see that they are handling RJ45 ports